samples/matter/thermostat/sample.yaml

```diff
@@ -8,7 +8,6 @@ tests:
     integration_platforms:
       - nrf52840dk/nrf52840
       - nrf5340dk/nrf5340/cpuapp
-      - nrf7002dk/nrf5340/cpuapp
```

Comment:
Minor: probably in such case you should also remove nRF7002dk from the platform_allow: few rows below.